test/e2e/e2e_test.go

```diff
@@ -74,6 +76,19 @@ func createServingCertAnnotatedService(client *kubernetes.Clientset, secretName,
 	return err
 }
 
+func createAnnotatedCABundleInjectionConfigMap(client *kubernetes.Clientset, configMapName, namespace string) error {
+	_, err := client.CoreV1().ConfigMaps(namespace).Create(&v1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: configMapName,
+			Annotations: map[string]string{
+				api.AlphaInjectCABundleAnnotationName: "true",
```

Comment:
Shouldn't we be testing this with the current version now? Or maybe with both?

Reply:
dunno, we'll clear that up b4 merging though